### PR TITLE
Fix #1329: [AIX] disable some functions based on availability in libperfstat

### LINKED ISSUE #1329: psutil-5.4.7 Failes to install on AIX 6.1
I am trying to install psutil 5.4.7 on AIX6.1.There is no pip.
 Python2.7.6
`python setup.py install`
error info:
`"psutil/_psutil_aix.c", lin 246.24: 1506-275 (S) Unexpected text procinfo encountered.`
`"psutil/_psutil_aix.c", lin 246.5: 1506-045 (S) Undeclared identifier perfstat_process_t.`
`"psutil/_psutil_aix.c", lin 225.33: 1506-045 (S) Undeclared identifier procinfo.`
`"psutil/_psutil_aix.c", lin 519.22: 1506-022 (S) "if_iqdrops: is not a member of "struct {...}.`
`"psutil/_psutil_aix.c", lin 519.22: 1506-022 (S) "xmitdrops: is not a member of "struct {...}.`

Does anyone have the same problem?

### PR DESCRIPTION
**Pull request by wiggin15**
See #1329

### CODE REVIEW

File: psutil/tests/__init__.py [hunk 1]
```diff
@@ -160,21 +160,22 @@
...
+HAS_NET_IO_COUNTERS = hasattr(psutil, "net_io_counters")
 HAS_PROC_CPU_NUM = hasattr(psutil.Process, "cpu_num")
+HAS_PROC_IO_COUNTERS = hasattr(psutil.Process, "io_counters")
 HAS_RLIMIT = hasattr(psutil.Process, "rlimit")
-HAS_THREADS = hasattr(psutil.Process, "threads")
 HAS_SENSORS_BATTERY = hasattr(psutil, "sensors_battery")
 HAS_BATTERY = HAS_SENSORS_BATTERY and bool(psutil.sensors_battery())
 HAS_SENSORS_FANS = hasattr(psutil, "sensors_fans")
 HAS_SENSORS_TEMPERATURES = hasattr(psutil, "sensors_temperatures")
+HAS_THREADS = hasattr(psutil.Process, "threads")
```
Comment (giampaolo) [inline psutil/tests/__init__.py]: nice (the sorting order) =)

**Comment by zhujing3** [comment]
thx ! i have compiled this on aix6-tl7-sp4

**Comment by giampaolo** [comment]
Damn, I forgot to include this into 5.6.0. :(

**Comment by giampaolo** [comment]
5.6.1 including this fix is out.

**Pull request merged**